packages/devtools_app/lib/src/logging/flutter/logging_screen.dart

```diff
@@ -230,38 +232,37 @@ class _LogDetailsState extends State<LogDetails>
   @override
   Widget build(BuildContext context) {
     return Container(
-      color: Theme.of(context).cardColor,
       child: _buildContent(context, widget.log),
     );
   }
 
   Widget _buildContent(BuildContext context, LogData log) {
-    if (log == null) return const SizedBox();
-    if (log.needsComputing) {
-      return const Center(child: CircularProgressIndicator());
+    if (showInspector(log)) {
+      return _buildInspector(context, log);
+    } else {
+      return Stack(
+        children: [
+          _buildSimpleLog(context, log),
+          if (log != null && log.needsComputing) ...[
```

Comment:
or

```
if (log != null && log.needsComputing)
  const Center(child: CircularProgressIndicator()),
```

which uses slightly fewer bits of syntax.

Reply:
You're right, I just don't like ifs without braces :P

Reply:
Done!